doc/connectivity/networking/net_config_guide.rst

```diff
@@ -0,0 +1,131 @@
+.. _network_configuration_guide:
+
+Network Configuration Guide
+###########################
+
+.. contents::
+    :local:
+    :depth: 2
+
+This document describes how various network configuration options can be
+set according to available resources in the system.
+
+.. csv-table:: network configuration options
```

Comment:
a ``list-table`` would be easier to author / maintain going forward, and would not necessitate the use of double quotes